dbfit-java/build.gradle

```diff
@@ -294,6 +308,76 @@ project('netezza') {
         compile project(':dbfit-java:core')
         runtime 'org.netezza:nzjdbc3:7.2@jar'
     }
+
+    task connectionProperties {
+        doLast {
+            ext {
+                propertiesFile = 'TestDbConnectionDbFitNetezza.properties'
+                jdbcDriverClass = 'org.netezza.Driver'
+                jdbcUrlPrefix = 'jdbc:netezza://'
+                delimiterType = 'row'
+            }
+
+            def props = new Properties()
+            file(propertiesFile).withInputStream { props.load(it) }
+            ext.userName = props.getProperty('username')
+            ext.passWord = props.getProperty('password')
+            def propService = props.getProperty('service')
+            def propServiceParts = propService.tokenize(':')
+            def propServiceHost = propServiceParts[0]
+            def propServicePort = propServiceParts[1]
+            def propDatabase = props.getProperty('database')
+
+            ext.database = propDatabase.isEmpty() ? ('/' + propDatabase) : ''
+            ext.jdbcUrl = jdbcUrlPrefix + propServiceHost + (propServicePort.isEmpty() ? (':' + propServicePort) : '')
+        }
+    }
+
+    integrationTest.dependsOn connectionProperties
+
+    integrationTest.doFirst {
+        copy {
+            from connectionProperties.propertiesFile + '.default'
+            into '.'
+            rename '(.*).default', '$1'
+        }
+
+        copy {
```

Comment:
I just realized we're already doing something with the other databases. Maybe it's a more general thing to consider.